src/RuleSet/Sets/PhpCsFixerRiskySet.php

```diff
@@ -53,6 +53,7 @@ public function getRules(): array
             'php_unit_data_provider_return_type' => true,
             'php_unit_strict' => true,
             'php_unit_test_case_static_method_calls' => ['call_type' => 'self'],
+            'static_lambda' => true,
```

Comment:
would you consider any modification to ruleset definition to be DX only?
i would go with DX only for devs working on our repo, for devs using our tool for their repos, I am not that sure.

eg, imagine introduction of new ruleset, full redefinition of existing ruleset or even removal of ruleset - it doesn't sounds as DX to me

Reply:
local codebase changes extracted to https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7239

Reply:
Maybe we should introduce dedicated category for changes related to rulesets (just `ruleset`)? Because changes in rules are either fix, feat (new behavior) or refactor (changes in code without actual changes in logic). For rulesets the logic remains "the same", but scope of the process may be changed... Or maybe more generic `API` category for all changes related to how Fixer behaves on the users' side? Thinking out loud 😉.

Reply:
So far, we were counting them as MINOR releases, so I guess `feat`

Reply:
[I said it before](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7037#issuecomment-1581366355): Conventional Commits explicitly defines only `feat` and `fix` and allows any other type of commit, so all of the additional types in our current config are custom ones and we can modify this set of types as much as we want. The goal of CC is to have machine-readable pattern, it's not closed specification.